docs/applications/advanced/external-models.md

```diff
@@ -0,0 +1,29 @@
+# Importing External Models
```

Comment:
Can you also update the docs for the model config params (`applications/resources/models.md`)?